src/page-builder.ts

```diff
@@ -538,7 +538,8 @@ export class PageBuilder {
     const engine = this.pod.engines.getEngineByFilename(
       this.partialPaths.view
     ) as TemplateEngineComponent;
-    partialBuilder.push('<page-module>');
+    const htmlId = partial.id ? ` id="${partial.id}"` : '';
+    partialBuilder.push(`<page-module${htmlId}>`);
```

Comment:
I realized that by setting the `id` here we override the `id` set by the page inspector UI:

- https://github.com/blinkk/amagaki-plugin-page-builder/blob/f200b7328ab1095bf533c4d18837f62a3890894c/src/ui/page-module-inspector.ts#L22
- https://github.com/blinkk/amagaki-plugin-page-builder/blob/f200b7328ab1095bf533c4d18837f62a3890894c/src/ui/page-module-inspector.ts#L41

Could you modify the `elementId` there to use the preset ID (if one exists) and otherwise assign it an ID?

This is used to allow people to deep link to the numbered modules in staging:

https://github.com/blinkk/amagaki-plugin-page-builder/blob/f200b7328ab1095bf533c4d18837f62a3890894c/src/ui/page-module-inspector.ts#L80

Reply:
Done.